src/transformState.js

```diff
@@ -4,8 +4,27 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    }
+
+    if (action.type === 'removeProperties') {
+      for (const i of action.keysToRemove) {
```

Comment:
Let's create more meaningful variable name
```suggestion
      for (const key of action.keysToRemove) {
```